### DISCUSSION TITLE
Using an action to route to a custom page

### DISCUSSION THREAD

**Discussion by vv-ac** [filamentphp/filament#2282] [Help] [answered]
I know I am slow-witted here.
I have a resource table with a custom LinkAction that I want to use to call a custom page like this
```
                LinkAction::make('Manage')
                    ->url(fn (Testelement $record): string => route('manage', $record))
                    ->icon('heroicon-o-adjustments')
                    ->color('primary'),
```

for getPages() I added:
```
            'manage' => Pages\ManageTestelement::route('/manage'),
```

Refrehsing the page it tells me, Laravel does not know about the route.
Isn't getPages used to register the route incl. ModelBinding`?

I appreciate every bit of help here.
Thanks!

**Comment by danharrin** [comment, marked as answer]
`->url(fn (Testelement $record): string => ResourceName::getUrl('manage', ['record' => $record]))`

`route()` is just a Laravel helper that generates a URL from a named route. Not a specific resource route.

**Reply by danharrin** [reply]
Also, you need to pass `/{record}` to the `getPages()` route to access the bound record

**Reply by vv-ac** [reply]
I was SO close, I just missed the suffix "Resource" after "Testelement".
Thanks for supporting again here!

Works!
